Zephyr.podspec

```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target      = '11.0'
   s.tvos.deployment_target     = '11.0'
   s.watchos.deployment_target  = '9.0'
+  s.visionos.deployment_target  = '1.0'
```

Comment:
Unsure if I need to do anything else to support visionOS in the podspec – for example bumping some other version somewhere.